css/chat.css

```diff
@@ -330,6 +333,7 @@ html {
   line-height: 1.1;
   margin-top: 4px;
   color: rgba(51, 51, 51, 0.5);
+  /* Does it look food where -webkit-line-clamp is not supported */
```

Comment:
We have line clamp on other widget (I believe the old list widgets) and we have never had issues of incompatibility